lib/Dialect/FIRRTL/FIRRTLOps.cpp

```diff
@@ -3006,22 +3007,22 @@ Attribute AggregateConstantOp::getAttributeFromFieldID(uint64_t fieldID) {
 }
 
 LogicalResult BundleCreateOp::verify() {
-  if (getType().getNumElements() != getFields().size())
+  if (getType().get().getNumElements() != getFields().size())
     return emitOpError("number of fields doesn't match type");
-  for (size_t i = 0; i < getType().getNumElements(); ++i)
-    if (!areTypesConstCastable(getType().getElementTypePreservingConst(i),
+  for (size_t i = 0; i < getType().get().getNumElements(); ++i)
+    if (!areTypesConstCastable(getType().get().getElementTypePreservingConst(i),
                                getOperand(i).getType().cast<FIRRTLBaseType>()))
```

Comment:
Why does this strip one side but not the other? Haven't chased it down to say it's wrong but isn't obvious looking at this.